Mathlib/Analysis/InnerProductSpace/PiL2.lean

```diff
@@ -134,23 +134,23 @@ def EuclideanSpace.delabVecNotation : Delab :=
 
 end Notation
 
-theorem EuclideanSpace.nnnorm_eq {𝕜 : Type*} [RCLike 𝕜] {n : Type*} [Fintype n]
+theorem EuclideanSpace.nnnorm_eq {𝕜 : Type*} [NormedRing 𝕜] {n : Type*} [Fintype n]
```

Comment:
Here and the ones below: `EuclideanSpace` is only intended to be used for `RCLike 𝕜`. If someone wants this for a `NormedRing 𝕜`, then they should use `PiLp 𝕜 (n → 𝕜)` (or whatever is the correct syntax).